sample/Source/SentryPlayground/SentryPlaygroundGameInstance.cpp

```diff
@@ -17,9 +17,9 @@ void USentryPlaygroundGameInstance::Init()
 
 	const TCHAR* CommandLine = FCommandLine::Get();
 
-	// Check '-SentryIntegrationTest' commandline argument to decide between running integration tests
+	// Check '-integration-test' commandline argument to decide between running integration tests
 	// or launching the sample app with UI for manual testing
-	if (FParse::Param(FCommandLine::Get(), TEXT("SentryIntegrationTest")))
+	if (FParse::Param(FCommandLine::Get(), TEXT("integration-test")))
```

Comment:
maybe run this if there's any arg? For normal startup with UI, there would be none, right?

Reply:
Actually I see in the README.md that's not true...

so maybe check if the arg is one of the expected ones (crash-capture/message-capture)

Where I'm coming from is trying to avoid adding additional args and differences between the gdx repos when we can avoid this

Reply:
IIRC Unity does exactly this - if it matches any of the expected args, it does the thing, otherwise it shows the UI